HelloWorldServlet/src/example/HelloWorld.java

```diff
@@ -0,0 +1,54 @@
+package example;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class HelloWorld
+ */
+@WebServlet("/SimplExample")//servlet url
+public class HelloWorld extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    public HelloWorld() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+		response.setContentType("text/html");
+		PrintWriter printWriter = response.getWriter();
```

Comment:
Object `PrinterWriter` extends from a closed interface, and it should be close after use. In my opinion, it's a good idea to close it - `try(..close..) {some code}`. More info about `try(...close...)` : https://www.baeldung.com/java-try-with-resources